src/Tribe/Aggregator/Record/Url.php

```diff
@@ -16,4 +16,37 @@ class Tribe__Events__Aggregator__Record__Url extends Tribe__Events__Aggregator__
 	public function get_label() {
 		return __( 'Other URL', 'the-events-calendar' );
 	}
+
+	/**
+	 * Matches which other origin this source url might be
+	 *
+	 * @since  TBD
+	 *
+	 * @param  string $source Which source we are testing against
+	 *
+	 * @return string|bool
```

Comment:
I think it might be more consistent easier if this one always returns a string and when is `false` returns an empty string instead.

Reply:
That actually strikes as odd, I mean, it does make sense to simplify the results but it also empty string isn't the clearer "answer" in my view for what just happened.

Reply:
I think you make a good point. ;)